standard.md

```diff
@@ -0,0 +1,272 @@
+
+### `id`
+
+Any change in `id` is **Non-Breaking**
+
+### `defaultContentType`
+
+Change in `defaultContentType` is **Breaking**
+
+### `info`
+
+**Breaking:**
+
+1. Change in `version`
+1. Change in `termsOfService`
+1. Change in `license` field
+
+**Non-Breaking:**
+
+1. Change in `title`
+1. Change in `description`
+1. Change in `contact` field
+
+### `servers`
+
+**Breaking:**
+
+1. Removal of a server(equivalent to change in server name)
+1. Change in `url`
+1. Change in `protocol`
+1. Change in `protocolVersion`
+1. Change in `default` (`variables`)
+1. Removal in `enum` (`variables`)
+1. Change in `security`
+
+**Non-Breaking:**
+
+1. Server being added
+1. Change in `description`
+1. Change in `description` (`variables`)
+1. Change in `examples` (`variables`)
+1. Addition in `enum` (`variables`)
+
+### `channels`
+
+**Breaking:**
+
+1.  Removal of channel
+1.  Removal of `subscribe` or `publish` field
+1.  Change in value of `$ref`
+
+**Non-Breaking:**
+
+1. Addition of a channel
+1. Change in `description`
+1. Addition of `subscribe` or `publish` field
+
+#### `subscribe` | `publish`
+
+**Breaking:**
+
+1. Change in `operationId`
+1. Change in `traits`
+
+**Non-Breaking:**
+
+1. Change in `summary`
+1. Change in `description`
+1. Change in `externalDocs`
+1. Change in `tags`
+
+##### `message`
+
+**Breaking:**
+
+1. Change in `contentType`
+1. Change in `name`
+1. Change in `payload`
+1. Change in `traits`
+1. Change in `headers`
+
+**Non-Breaking:**
+
+1. Change in `summary`
+1. Change in `description`
+1. Change in `examples`
+1. Change in `title`
+1. Change in `tags`
+1. Change in `externalDocs`
+1. Change in `correlationId`
+
+#### `parameters`
+
+**Breaking:**
+
+1. Change in `schema`
+1. Change in `location`
+1. Change in value of `$ref`
+
+**Non-Breaking:**
+
+1. Change in `description`
+
+### `components`
+
+Any change in `components` is **non-breaking**.
+
+**Reason:** The parser will inline the `$ref` during the parsing phase.
+
+### `tags`
+
+Any change in `tags` is **non-breaking**.
+
+### `externalDocs`
+
+Any change in `externalDocs` is **non-breaking**.
+
+## Summary
+
+### Breaking changes
+
+1. Change in `major` version in `asycnapi`
+1. Change in `defaultContentType` field
+1. Change in `version` in `info`
+1. Change in `termsOfService` in `info`
+1. Change in `license` field in `info`
+1. Removal of a server(equivalent to change in server name)
+1. Change in `url` in `servers`
+1. Change in `protocol` in `servers`
+1. Change in `protocolVersion` in `servers`
+1. Change in `default` (`variables`) in `servers`
+1. Removal in `enum` (`variables`) in `servers`
+1. Change in `security` in `servers`
+1.  Removal of channel
+1.  Removal of `subscribe` or `publish` field in `channels`
+1.  Change in value of `$ref` in `channels`
+1. Change in `operationId` in `subscribe`|`publish`
+1. Change in `traits` in `subscribe`|`publish`
+1. Change in `contentType` in `message`
+1. Change in `name` in `message`
```

Comment:
> From the POV of a user, the change in API will be a breaking change (if it affects the way user interacts with the API).

one could argue it is not a breaking change, name of a message is not much important

Reply:
ah, okay. Will consider it a non-breaking change then. :+1: